pxr/usd/usdLux/overview.dox

```diff
@@ -101,11 +101,60 @@ constraints, such as to attach lights to moving geometry.  This is
 consistent with USD's policy of storing the computed result of rigging,
 not the rigging itself.
 
+\subsection usdLux_quantities Quantities and Units
+
+Most renderers consuming OpenUSD today are RGB renderers, rather than spectral.
+Units in RGB renderers are tricky to define as each of the red, green and blue
+channels transported by the renderer represents the convolution of a spectral
+exposure distribution, e.g. CIE Illuminant D65, with a sensor response function,
+e.g. CIE 1931 𝓍̅. Thus the main quantity in an RGB renderer is neither radiance
+nor luminance, but "integrated radiance" or "tristimulus weight".
+
+The emission of a default light with `intensity` 1 and `color` [1, 1, 1] is an
+Illuminant D spectral power distribution with chromaticity matching the
```

Comment:
`spectral distribution` ~ the word power isn't helping clarity

Reply:
Done: https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/2533cb0a9bbfed73a5e2c970c67d8b2f91a27b52#diff-bdeeccf2a8491e83067bc3c1e20e5a696612da9b31d1e3b7e6e7fc33b3592ec7R67